testing/jormungandr-testing-utils/src/testing/fragments/mod.rs

```diff
@@ -105,14 +99,35 @@ impl FragmentBuilder {
         addresses: &[Address],
         value: Value,
     ) -> Result<Fragment, FragmentBuilderError> {
-        transaction_to_many(
-            &self.block0_hash,
-            &self.fees,
-            self.valid_until,
-            from,
-            addresses,
-            value,
-        )
+        let mut iobuilder = InputOutputBuilder::empty();
+
+        for address in addresses {
+            iobuilder
+                .add_output(address.clone().into(), value.into())
+                .unwrap();
+        }
+
+        let value_u64: u64 = value.into();
+        let input_without_fees: Value = (value_u64 * addresses.len() as u64).into();
+        let input_value = self
+            .fragment_factory
+            .fee
+            .calculate(None, 1, addresses.len() as u8)
+            + input_without_fees.into();
+        let input = from.add_input_with_value(input_value.unwrap().into());
```

Comment:
this only works for accounts, not sure if we allowed utxos before / we are interested in providing this service for utxos wallets. If that's not the case I suggest to restrict the input type to `account::Wallet`

Reply:
it was just a copy paste. `add_input_with_value` will panic for utxo. Is that ok, or should i signal it in method name that it is only account based operation?

Reply:
if we accept in input a `account::Wallet` instead of a general purpose wallet it's enough, even though signaling this in the name would help